src/scripts/apploader.js

```diff
@@ -1,52 +1,48 @@
-(function() {
```

Comment:
I think we should keep the closure here so we aren't unnecessarily polluting the global scope.

Reply:
It's going to be replaced by Webpack soon anyway, so it was mostly to make it clearer at a glance and because I hate anonymous functions like this :p

Reply:
But it serves a purpose until that change is actually in place :wink:

Reply:
Fine, fixed it :)